### PR TITLE
feat：新增 v2查询分级管理员下用户组接口 #6

### PR DESCRIPTION
**Pull request by fcfang123**
feat：新增 v2查询分级管理员下用户组接口 #6

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/iam-java-sdk?pullRequest=7) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/iam-java-sdk?pullRequest=7) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/iam-java-sdk?pullRequest=7) it.</sub>

File: src/main/java/com/tencent/bk/sdk/iam/service/v2/impl/V2ManagerServiceImpl.java [hunk 1]
```diff
@@ -471,10 +470,28 @@ public V2ManagerRoleGroupVO getGradeManagerRoleGroupV2(String gradeManagerId, St
         return null;
     }
 
+    private String buildSearchGroupUrl(SearchGroupDTO searchGroupDTO, String url) {
+        if (searchGroupDTO.getInherit() != null)
```
Comment (mingshewhe) [inline src/main/java/com/tencent/bk/sdk/iam/service/v2/impl/V2ManagerServiceImpl.java:474]: 这里抽象成通用方法，由bean转换成查询url

File: build.gradle [hunk 2]
```diff
@@ -64,6 +64,8 @@ dependencies {
     // https://mvnrepository.com/artifact/org.slf4j/slf4j-api
     implementation(group: 'org.slf4j', name: 'slf4j-api', version: '1.7.30')
 
+    implementation(group: 'com.alibaba', name: 'fastjson', version: '1.2.47')
```
Comment (mingshewhe) [inline build.gradle:67]: 不要用fastjson，工程中已经有json依赖包

File: src/main/java/com/tencent/bk/sdk/iam/util/OkhttpUtils.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class OkhttpUtils {
+
+    public static String joinParams(Map<String, String> params) throws UnsupportedEncodingException {
```
Comment (mingshewhe) [inline src/main/java/com/tencent/bk/sdk/iam/util/OkhttpUtils.java:15]: 不要新建一个工具类，可以在HttpClientService新增一个doHttpPost方法,可以指定params

**Review by mingshewhe — APPROVED** [review]

**Pull request merged**